appinventor/docs/reference/components/media.html

```diff
@@ -651,8 +651,18 @@ <h2 id="Sound">Sound</h2>
                   <h2 id="SoundRecorder"> SoundRecorder </h2>
                     <img src="images/soundRecorder.png" alt="Picture of Sound Recorder component">
                     <p> Multimedia component that records audio. </p>
-                    <h3> Properties </h3>
-                    none
+		    <h3>Properties</h3>
+		    <dl>
+		      <dt><code>SavedRecording</code></dt>
+		      <dd>Specifies the path to the file where the
+		      recording should be stored. If this proprety is
+		      the empty string, then starting a recording will
+		      create a file in an appropriate location.  If
+		      the property is not the emppty string, it should
```

Comment:
thanks :)



== Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.
MIT Media Lab, Second Class Faculty


On Mon, Aug 10, 2015 at 11:01 AM, Jeffrey Schiller <notifications@github.com
> wrote:

> In appinventor/docs/reference/components/media.html
> <https://github.com/mit-cml/appinventor-sources/pull/497#discussion_r36640875>
> :
>
> > @@ -651,8 +651,18 @@ <h2 id="Sound">Sound</h2>
> >                    <h2 id="SoundRecorder"> SoundRecorder </h2>
> >                      <img src="images/soundRecorder.png" alt="Picture of Sound Recorder component">
> >                      <p> Multimedia component that records audio. </p>
> > -                    <h3> Properties </h3>
> > -                    none
> > +		    <h3>Properties</h3>
> > +		    <dl>
> > +		      <dt><code>SavedRecording</code></dt>
> > +		      <dd>Specifies the path to the file where the
> > +		      recording should be stored. If this proprety is
> > +		      the empty string, then starting a recording will
> > +		      create a file in an appropriate location.  If
> > +		      the property is not the emppty string, it should
>
> Here too... :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-cml/appinventor-sources/pull/497/files#r36640875>.
>